### PR TITLE
LOK-2487: Alert Rule Table Display fixes

### PR DESCRIPTION
**Pull request by Syed-Bilal-Haider-Dataq**
## Description
Alert rule table displays, wherever they appear, should have following columns:

Name - name of the rule

Type - event type, friendly name, e.g. “Internal”, “SNMP Trap”, “Syslog”, “Metric Threshold”

Description - for now let's use Severity / Trigger Event name. For example: “Critical / SNMP Cold Start”

## Jira link(s)
https://opennms.atlassian.net/browse/LOK-2487

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

**Review by synqotik — CHANGES_REQUESTED** [review]
I updated LOK-2487, please take a look. Note, you also need to fix the table header names.

File: ui/src/components/MonitoringPolicies/MonitoringPoliciesAlertsRulesTable.vue [hunk 1]
```diff
@@ -85,8 +85,8 @@ const deleteMsg = computed(() =>
 )
 
 const columns: { id: string; label: string }[] = [
-  { id: 'AlertRule', label: 'Alert Rule' },
-  { id: 'ConditionalLogic', label: 'Conditional Logic' },
+  { id: 'NameOdRule', label: 'Name' },
```
Comment (synqotik) [inline ui/src/components/MonitoringPolicies/MonitoringPoliciesAlertsRulesTable.vue:88]: Should be `NameOfRule` or just `Name`

**Review by synqotik — APPROVED** [review]
One typo to fix before merging, other than that looks good.

File: ui/src/components/MonitoringPolicies/MonitoringPoliciesAlertsRulesTable.vue (hunk 1 above)
Comment (Syed-Bilal-Haider-Dataq) [reply ui/src/components/MonitoringPolicies/MonitoringPoliciesAlertsRulesTable.vue]: Oh. I check it.

**Pull request merged**